rezolvari probleme.js

```diff
@@ -0,0 +1,71 @@
+//problema 1
+function SumVect(vect)
+{
+    var sum = 0;
+
+    for(i=0; i < vect.length; i++)
+    {
+        sum += vect[i];
+    }
+
+    return sum;
+}
```

Comment:
I think you missed a part of the problem here: ``an integer array of size N, compute the sum of all  even numbers in this array.`` Key word here is **even**